index.html

```diff
@@ -362,6 +370,6 @@ <h1 class="Contactme">Contact Me</h1>
         </form>
         
       </footer>
-    </section>
+    
```

Comment:
- [ ] The About section background is getting stretched, as you can see in the upper right corner, It would be better to avoid distorting it:

|Template|Your design|
|---|---|
|![Captura de pantalla 2022-06-01 a la(s) 9 37 44 p  m](https://user-images.githubusercontent.com/83717285/171541177-48472059-f83d-468b-96d2-ddc8a93247ac.png)|![Captura de pantalla 2022-06-01 a la(s) 9 38 03 p  m](https://user-images.githubusercontent.com/83717285/171541239-15201bd5-f5ef-463d-ad58-11a56cd82dd9.png)|

Reply:
done